### ISSUE TITLE
Mask-RCNN and SSD bounding boxes on H36M

### ISSUE THREAD

**Issue by Billccx** [karfly/learnable-triangulation-pytorch#159]
Hi!
Thanks for your research contribution. 
I want to use Mask-RCNN or SSD bounding boxes instead of H36M's ground truth. Would you please share the following files?
```
detections_paths = {
      'MRCNN': {
          'train': "/Vol1/dbstore/datasets/Human3.6M/extra/train_human36m_MRCNN.json",
          'test': "/Vol1/dbstore/datasets/Human3.6M/extra/test_human36m_MRCNN.json"
      },
      'SSD': {
          'train': "/Vol1/dbstore/datasets/k.iskakov/share/ssd-detections-train-human36m.json",
          'test': "/Vol1/dbstore/datasets/k.iskakov/share/ssd-detections-human36m.json"
      }
  }
```
I am looking forward to your reply and thanks in advance!

**Comment by karfly** [comment]
Unfortunately, these files are lost. We are unable to recover them, as we no longer work at Samsung
